components/aws/sagemaker/unit_tests/run_all_tests.sh

```diff
@@ -0,0 +1,4 @@
+export PYTHONPATH=../
+
+coverage run -m pytest --junitxml ./log
```

Comment:
nit: Name of the report should be more descriptive than `log`.

Reply:
done 
changed to "unit_tests_log"